source/docs/Motors/servo-motors.rst

```diff
@@ -156,31 +156,57 @@ Standard Servo
          //Include the Servo Library
          #include "Servo_ros.h"
          
-         /**
-          * Constructor
-          * Servo's ros threads (publishers and services) will run asynchronously in the background
-          */
-          
-         ros::NodeHandle nh; //internal reference to the ROS node that the program will use to interact with the ROS system
-         VMXPi vmx(true, (uint8_t)50); //realtime bool and the update rate to use for the VMXPi AHRS/IMU interface, default is 50hz within a valid range of 4-200Hz
          
-         ServoRos servo(&nh, &vmx, channel);
-         ros::ServiceClient setAngle;
+         double servo_angle;
+            
+         void servo_angle_Cb(const std_msgs::Float32::ConstPtr& msg)
```

Comment:
the function name is confusing. Also, needs a comment block? (What does this specific function do)